docs/source/api.rst

```diff
@@ -120,3 +120,17 @@ Tests to compare whether one forecast is significantly better than another one.
 
     sign_test
     mae_test
+
+Resampling
+----------
+
+Resampling functions to create a new ``iteration`` dimension based on one dimension
+with or without replacement.
+
+.. currentmodule:: xskillscore.resampling
```

Comment:
It must not like this. So maybe my idea was wrong. You probably need to say `xskillscore.core.resampling` here and the warnings will go away.